remix-analyzer/src/solidity-analyzer/modules/gasCosts.ts

```diff
@@ -1,40 +1,35 @@
 import { default as category } from './categories'
 import { default as algorithm } from './algorithmCategories'
-import { ModuleAlgorithm, ModuleCategory, ReportObj, CompilationResult} from './../../types'
+import  AbstractAst from './abstractAstView'
+import { ModuleAlgorithm, ModuleCategory, ReportObj, CompilationResult, CompiledContractObj, CompiledContract, VisitFunction, AnalyzerModule} from './../../types'
 
-export default class gasCosts {
+type VisitedContract = {
```

Comment:
Use interface when you can instead of type. It's easier to read on VSCode tooltip as interface types are display as an object & interface with their name.